canonical-websites/www.ubuntu.com
# Add modal contact us form to the IoT section of the site

## Done
Add modal contact us form to the IoT section of the site

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Go to the /core page
- In the console run `dynamicContactUs()`
- Then click through the interactive form and check you get the right thank you page
- Go to the /internet-of-things page
- In the console run `dynamicContactUs()`
- Then click through the interactive form and check you get the right thank you page
- Repeat on the other pages of the section

## Issue / Card
Fixes https://github.com/canonical-websites/www.ubuntu.com/issues/4575
